apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.ts

```diff
@@ -299,7 +300,26 @@ router.post(
         req.body.qid,
       );
 
-      res.redirect(res.locals.urlPrefix + '/question/' + qid + '/settings');
+      const client = getCourseFilesClient();
+
+      const result = await client.batchDeleteQuestions.mutate({
+        course_id: res.locals.course.id,
+        user_id: res.locals.user.user_id,
+        authn_user_id: res.locals.authn_user.user_id,
+        has_course_permission_edit: res.locals.authz_data.has_course_permission_edit,
+        question_ids: [question.id],
+      });
+
+      if (result.status === 'error') {
+        throw new error.HttpStatusError(
+          500,
+          'Draft deletion failed, but question creation succeeded.',
+        );
+      }
+
+      flash('success', 'Succesfully finalized question.');
```

Comment:
Do we have access to the QID here? Could we say "Successfully finalized question: now available as ${QID}"?
Also typo:
```suggestion
      flash('success', 'Successfully finalized question.');
```

Reply:
Done in 73911c7.

Reply:
Maybe style the QID somehow? Bold or code styling? @nwalters512 do we have a standard convention for this?